### DISCUSSION TITLE
**Added** the ability to set `credentialName` based secret configuration at sidecars for egress TLS traffic when `WorkloadSelector` is specified in `DestinationRule`, provided the sidecar has permission to list secrets in the namespace where it resides. I do not understand this？

### DISCUSSION THREAD

**Discussion by 13567436138** [istio/istio#49472] [Q&A]
**Added** the ability to set `credentialName` based secret configuration at sidecars for egress TLS traffic when `WorkloadSelector` is specified in `DestinationRule`, provided the sidecar has permission to list secrets in the namespace where it resides. I do not understand this？

**Comment by leosarra** [comment]
It means that, in case you have a DestinationRule which is scoped via workloadSelector to only apply to certain sidecars, then you can use the "credentialName" field of the DR such that a specific secret is used for egress mTLS/TLS traffic (if sidecar has permissions to list the secrets)
